### DISCUSSION TITLE
Quick note floating button or notification area quick note

### DISCUSSION THREAD

**Discussion by spc789** [damionx7/Zettel-Notes-Documentation#74] [Feature Requests]
Add a floating button (shazam like, over the other apps) button to open a quick note.
Or a notification area start quick note feature (like in orgzly) to spead up note taking

**Comment by damionx7** [comment]
You can pin your notes as notification. Read more https://znotes.thedoc.eu.org/note/share-note/notifications/.

Also you can create quick note tile.

**Comment by spc789** [comment]
Couod be you interpreted the second suggestion wrong.
I didn't mean a tile on the startscreen. I ment a button which is drawn over all the other apps

**Reply by damionx7** [reply]
Ok. I reopened this discussion.
